### PR TITLE
Backports 0.19 pr4

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

**Comment by Munkybooty** [comment]
I'm assuming it probably isn't a timeout issue since the linux 64 test failed after 2 minutes, right?

File: src/qt/forms/debugwindow.ui [hunk 1]
```diff
@@ -1395,6 +1395,19 @@
...
                </item>
+               <item row="20" column="0">
+                <spacer name="verticalSpacer_3">
+                 <property name="orientation">
+                  <enum>Qt::Vertical</enum>
+                 </property>
+                 <property name="sizeHint" stdset="0">
+                  <size>
+                   <width>20</width>
+                   <height>10</height>
```
Comment (PastaPastaPasta) [inline src/qt/forms/debugwindow.ui:1406]: no real objection, but upstream is 40, why the change?

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

File: src/qt/forms/debugwindow.ui (hunk 1 above)
Comment (UdjinM6) [reply src/qt/forms/debugwindow.ui:1406]: see https://github.com/dashpay/dash/pull/4543#discussion_r735677143: I compiled it and it behaved weird (imo) with 40 - there is enough free space and yet you see scrollbars

**Review by UdjinM6 — APPROVED** [review]
utACK

File: src/qt/forms/debugwindow.ui (hunk 1 above)
Comment (UdjinM6) [reply src/qt/forms/debugwindow.ui:1406]: Also, I have no idea why `UdjinM6 dismissed PastaPastaPasta’s stale review via 8179bd5` happened 🤷‍♂️  pls re-review @PastaPastaPasta

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit
